proprietary/tokens/src/brand/ams/typography.tokens.json

```diff
@@ -0,0 +1,39 @@
+{
+  "ams": {
+    "brand": {
+      "typography": {
+        "font-family": {
+          "body-text": { "value": "'Amsterdam Sans', Arial, sans-serif" },
+          "headings": { "value": "'Amsterdam Sans', Arial, sans-serif" },
```

Comment:
Headings include labels and fieldset legends as well. And accordion buttons use these tokens.